mx.pwmc.Svgvi.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id": "mx.pwmc.Svgvi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "writable-sdk": true,
+    "command": "mx.pwmc.Svgvi",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options": {
+        "cflags": "-O2 -g",
+        "cxxflags": "-O2 -g",
+        "env": {
+            "V": "1"
+        }
+    },
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+                "name": "libgee",
+                "builddir": true,
+                "sources": [
+                        {
+                                "type": "archive",
+                                "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.1.tar.xz",
+                                "sha256": "bb2802d29a518e8c6d2992884691f06ccfcc25792a5686178575c7111fea4630"
+                        }
+                ]
+        },
+        {
+                "name": "gxml",
+                "builddir": true,
+                "sources": [
+                        {
+                        "type": "archive",
+                        "url": "https://download.gnome.org/sources/gxml/0.16/gxml-0.16.3.tar.xz",
+                        "sha256": "520d4d779b1d31591762b2a98f84072531b9e17ac401df9668493e189eafc6ba"
+                        }
+                ]
+        },
+        {
+                "name": "gresg",
+                "buildsystem": "meson",
+                "builddir": true,
```

Comment:
It's already default for meson.

Reply:
(also applies below)

Reply:
`gresg` still have both meson and Autotools, until the last is removed I need to set the buildsystem to use.

Reply:
What I mean is that `"buildsystem": "meson"` turns `"builddir"` on.